media-libs/webvfx/webvfx-1.1.0.ebuild

```diff
@@ -0,0 +1,49 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit flag-o-matic qmake-utils toolchain-funcs
+
+DESCRIPTION="Video effects library based on web technologies"
+HOMEPAGE="https://github.com/mltframework/webvfx/"
+SRC_URI="https://github.com/mltframework/${PN}/releases/download/1.1.0/${P}.txz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
```

Comment:
Haven't really checked upstream build system, but do these files get generated and installed?
https://github.com/mltframework/webvfx/tree/master/doc
I'd say a perfect candidate for `doc` IUSE, and it seems to use `doxygen` to generate them which would be a BDEPEND.